source/apiVolontaria/location/factories.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+import factory
+import factory.fuzzy
+
+from .models import *
+
+
+class CountryFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Country
+
+    name = "random country"
+    iso_code = factory.fuzzy.FuzzyText(length=2)
```

Comment:
`Faker` have **lot of** functions you can use for this type of purpose:
```
fake.country_code()
# 'AG'
```

https://faker.readthedocs.io/en/latest/providers/faker.providers.address.html

---

But here you have a problem because the iso_code is unique so we will have some error